jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh

```diff
@@ -17,6 +17,7 @@ set -e -x
 #
 
 # The versions below don't matter; they are replaced by the Jenkins job
+# TODO Would be nice to read the image URLs below from automation/src/test/resources/reference.conf
```

Comment:
This comment is a bit misleading because it already does this if "use explicit versions" is unchecked in the Jenkins job: it reads from reference.conf and replaces these strings with `sed`. 

@Qi77Qi's comment above would be a nice future enhancement: to all _specific_ image overrides, while using `reference.conf` as a default. But that's out of scope here, and I don't think it warrants a TODO.

Reply:
Thanks. Yea, I realize I'd missed the fact that automation `reference.conf` is already where the versions are extracted from and used when manual overrides aren't set.